randovania/games/prime1/exporter/game_exporter.py

```diff
@@ -94,7 +94,9 @@ def wrap_text(text):
 
     for area in region.areas:
         for node in area.nodes:
-            if not isinstance(node, DockNode):
+            is_dock_node = isinstance(node, DockNode)
+            telporter_node = is_dock_node and node.dock_type.short_name not in {"teleporter"}
+            if not is_dock_node or telporter_node:
```

Comment:
I think this need to be `in` instead of `not in`. Skip for teleporter because they don't have that dock_index thing

Reply:
Maybe skipping anything without a dock_index is best. Though I'd say @toasterparty might have ideas

Reply:
Now, I ignore all docks which are teleporter. This is equivalent to the logic before.